README.rst

```diff
@@ -5,6 +5,10 @@ An agglomeration of integration tests for the `LLVM Toolchain <https://llvm.org/
 Unlike the various existing llvm testing suites, this is meant to be used in an installed
 environment to detect packaging issues.
 
+This is an initiative started by Debian/Ubuntu and Fedora to agregate the testing effort.
```

Comment:
stated by Debian/Ubuntu and Fedora *maintainers*, because I don't act on behalf of the Fedora organization.